tst/teststandard/opers/IsomorphismGroups.tst

```diff
@@ -10,7 +10,7 @@ gap> Length(CharacteristicSubgroups(g));
 5
 gap> Length(CharacteristicSubgroups(h));
 5
-gap> IsomorphismGroups(g,h:forcetest)<>fail;
+gap> IsomorphismGroups(g,h)<>fail;
```

Comment:
The `forcetest` option makes the algorithm immediately go into a costly step of applying the p-quotient algorithm to what will be a Sylow subgroup (so that that code will be tested). By default this only happens much later if the (faster) standard heuristics do not make progress.
The hope was that this way the default test would have checked that code, alas it turned out that travis timed out (and I had to remove the test again).